foreshadow/concrete/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""All the concrete transformers provided by foreshadow."""
+
+from foreshadow.concrete.externals import *  # noqa: F403, F401
+from foreshadow.concrete.internals import *  # noqa: F403, F401
+
+
+__all__ = [str(s) for s in globals()]
```

Comment:
Did we end up figuring out if the globals() here was causing the import bugs?

Reply:
well it works now so maybe, maybe not.

Reply:
On hindsight, I think this was causing the errors, specifically the use of globals().